src/app.ts

```diff
@@ -251,6 +256,8 @@ class App {
 
   getRemoteVideoStream = (call: Call) => Wazo.Phone.phone.getRemoteVideoStream(call);
 
+  updateBadge = (args: UpdateBadgeArgs) => this._sendMessage(EVENT_UPDATE_BADGE, args);
```

Comment:
I think that the SDK should send the `entityId` as it knows it and it will remove necessary work for the user.